core/shared/src/test/scala/scalaz/zio/TestUtils.scala

```diff
@@ -0,0 +1,16 @@
+package scalaz.zio
+
+trait TestUtils {
```

Comment:
My one suggestion is that maybe this should be in the `TestRuntime` directly, rather than having a new file for it? No strong preference either way, though

Reply:
🤦‍♂ of course